From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Thu, 31 Oct 2019 10:40:13 -0700
Subject: ANDROID: Remove KVM_INTEL allmodconfig workaround

This was blocked on upgrading clang for asm goto support, we can
re-enable this working driver.

Bug: 140224784
Change-Id: I715c3611b2aff2b8a898ae85e45cce1dc149deef
Signed-off-by: Alistair Delva <adelva@google.com>
---
 build.config.allmodconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build.config.allmodconfig b/build.config.allmodconfig
index f91c03b85737..b39bc1b877c7 100644
--- a/build.config.allmodconfig
+++ b/build.config.allmodconfig
@@ -9,7 +9,6 @@ function update_config() {
          -d TEST_KMOD  \
          -d XFS_FS \
          -d CPU_BIG_ENDIAN \
-         -d KVM_INTEL \
          -d STM \
          -d TEST_MEMCAT_P \
 
